### PR TITLE
add profiling options to nix and cabal files of brig and galley

### PR DESCRIPTION
**Pull request by MangoIV**
## Checklist

 - [ ] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [x] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

**Comment by supersven** [comment]
To enable the external storage on EKS-based clusters, this needs to be set in the deployment's Helm values of `brig` and `galley`:

```yaml
profLogs:
  enabled: true
  storageClassName: gp2
``

File: charts/brig/templates/deployment.yaml [hunk 1]
```diff
@@ -167,3 +178,14 @@ spec:
...
+  {{- if .Values.profLogs.enabled }}
+  volumeClaimTemplates:
+    - metadata:
+        name: prof-logs
+      spec:
+        accessModes: [ "ReadWriteOnce" ]
+        storageClassName: {{ .Values.profLogs.storageClassName }}
+        resources:
+          requests:
+            storage: "10Gi"
```
Comment (elland) [inline charts/brig/templates/deployment.yaml:190]: I'm wondering if that's enough. I got close to 9gb in 12-13 hours of profiling. Maybe we do `50gi`?

Comment (MangoIV) [reply charts/brig/templates/deployment.yaml:190]: no. None of the available tools can handle event logs of > 2G graciously. If we go farther above we won’t get anything useful out of it. That’s why we increase the sample interval from .1 to 1 seconds

Comment (MangoIV) [reply charts/brig/templates/deployment.yaml:190]: Also your profile didn’t do the right thing, unfortunately, the ones that we need produce ~2GB per hour.

**Comment by MangoIV** [comment]
not planned

**Pull request closed without merging**